src/components/BuySell/BuySellDisabled.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import { TokenDataProps } from 'components/ProductPage/ProductDataUI'
+import useChainData from 'hooks/useChainData'
+
+const BuySellDisabled = (props: { tokenData: TokenDataProps }) => {
```

Comment:
Are we planning to support this in a follow on PR?

Reply:
we will now